utils/json-schemas/performance_notification.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://frinx.io/performance_notification.json",
+  "title": "Performance monitoring notification",
+  "description": "Schema for performance monitoring notifications",
+  "type": "object",
+  "properties": {
+    "node_id": {
+      "description": "The name of the node.",
+      "type": "string"
+    },
+    "identifier": {
+      "description": "Identifier/path to data.",
+      "type": "string"
+    },
+    "event_time": {
+      "description": "The timestamp when notification arrived.",
+      "type": "string"
```

Comment:
there are already some built-in timestamp formats: https://json-schema.org/understanding-json-schema/reference/string#dates-and-times

does it fit one of them?
if not, you can still define pattern-based constraint

Reply:
this is the timestamp from notification: